code/clue.js

```diff
@@ -7,53 +7,332 @@ const mrGreen = {
+  occupation: 'War profiteer',
+  favouriteWeapon: 'poison',
+}
+console.log(colonelMustard.description);
+
+const mrsWhite = {
+  firstName: 'Mrs',
+  lastName: 'White',
+  color: 'white',
+  description: 'A frazzled servant. Depicted as the widow of a nuclear physicist whose death she is suspected of being behind',
+  age: 63,
+  image: 'assets/white.png',
+  occupation: 'Nurse',
+  favouriteWeapon: 'axe',
+}
+console.log(mrsWhite.description);
 
 // CREATE OBJECTS FOR ALL THE WEAPONS, ADD MORE CHARACTERISTICS TO THE WEAPONS IF YOU LIKE.
 
 const rope = {
-  name: 'rope',
-  weight: 10
+  name: 'Rope',
+  material: 'textile',
+  id: 'rope',
+  image: 'assets/rope.png',
+}
+console.log(rope);
+
+const knife = {
+  name: 'Knife',
+  material: 'metal',
+  id: 'knife',
+  image: 'assets/knife.png',
+}
+console.log(knife);
+
+const candlestick = {
+  name: 'Candlestick',
+  material: 'gold',
+  id: 'candlestick',
+  image: 'assets/candlestick.png',
+}
+console.log(candlestick);
+
+const dumbbell = {
+  name: 'Dumbbell',
+  material: 'iron',
+  id: 'dumbbell',
+  image: 'assets/dumbbell.png',
+}
+console.log(dumbbell);
+
+const poison = {
+  name: 'Poison',
+  material: 'liquid',
+  id: 'poison',
+  image: 'assets/poison.png'
+}
+console.log(poison);
+
+const axe = {
+  name: 'Axe',
+  material: 'metal',
+  id: 'axe',
+  image: 'assets/axe.png',
+}
+console.log(axe);
+
+const bat = {
+  name: 'Bat',
+  material: 'wood',
+  image: 'assets/bat.png',
+}
+console.log(bat);
+
+const trophy = {
+  name: 'Trophy',
+  material: 'silver',
+  image: 'assets/trophy.png',
 }
+console.log(trophy);
+
+const pistol = {
+  name: 'Pistol',
+  material: 'metal',
+  image: 'assets/pistol.png',
+}
+console.log(pistol);
 
 // THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
 
 // NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
 
 const suspects = [
   mrGreen,
+  professorPlum,
+  missScarlet,
+  mrsPeacock,
+  colonelMustard,
   mrsWhite
-  // ...  and the rest
 ]
+console.log(suspects);
 
-const weapons = []
+const weapons = [
+  rope,
+  knife,
+  candlestick,
+  dumbbell,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol
+]
 
-const rooms = []
+const rooms = [
+  "Dining Room",
+  "Conservatory",
+  "Kitchen",
+  "Study",
+  "Library",
+  "Billiard Room",
+  "Lounge",
+  "Ballroom",
+  "Hall",
+  "Spa",
+  "Living Room",
+  "Observatory",
+  "Theater",
+  "Guest House",
+  "Patio"
+]
+console.log(rooms);
 
 // THIS FUNCTION WILL RANDOMLY SELECT ONE ITEM FROM THE ARRAY THAT YOU PASS IN TO THE FUNCTION.
 // YOU DON'T NEED TO CHANGE THIS, JUST TRY TO UNDERSTAND IT. AND HOW TO USE IT.
 const randomSelector = array => {
   return array[Math.floor(Math.random() * array.length)]
-}
+};
 
 // CREATE AN OBJECT THAT KEEPS THE MYSTERY.
-// With a killer, a weapon and a room.
-// The values will be set later.
+// With a killer, a weapon and a room, the values will be set later.
+let mystery = {
+  killer: '',
+  weapons: '',
+  rooms: '',
```

Comment:
It's even more semantic approach to initialise mystery object with null values like this :

```
const mystery = {
  killer: null,
  weapon: null,
  room: null
}
```